grow/commands/build.py

```diff
@@ -35,18 +36,20 @@ def build(pod_path, out_dir, preprocess, clear_cache, pod_paths, locate_untransl
     if locate_untranslated:
         pod.enable(pod.FEATURE_TRANSLATION_STATS)
     try:
-        config = local_destination.Config(out_dir=out_dir)
-        destination = local_destination.LocalDestination(config)
-        destination.pod = pod
-        paths, _ = pod.determine_paths_to_build(pod_paths=pod_paths)
-        repo = utils.get_git_repo(pod.root)
-        stats_obj = stats.Stats(pod, paths=paths)
-        content_generator = destination.dump(pod, pod_paths=pod_paths)
-        destination.deploy(content_generator, stats=stats_obj, repo=repo, confirm=False,
-                           test=False, is_partial=bool(pod_paths))
-        pod.podcache.write()
+        with pod.profile.timer('grow_build'):
```

Comment:
Hm I wonder if this can be built in to the top-level command?

Reply:
I thought about this, but it would require that we create the pod and place it in the context and not create it in each command. I'll test it out and see how well it works.

Reply:
I've dug into this idea and it doesn't quite work out. To make it work I would need to be able to access the arguments for the sub-command, which click is very explicit in not allowing. I've played around with a few ideas, but it has not worked.

The main issue right now is that we need to get access to the `pod_path` argument that is passed to the subcommand while in the main command. But that does not work for click and a couple of attempts at custom decorators has also not worked.